src/fava/ext/portfolio_list/PortfolioList.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  onReloadButtonClick() {
+    location.reload();
+  },
+  onExtensionInit() {},
```

Comment:
This extension should showcase some simple usecases of all the functions that we might call from the frontend - otherwise it is not really possible to test this functionality

Reply:
Yeah sorry, I have an extension that is using this functionality locally, but I couldn't come up with something general that would be useful as a demo. I'll try to add some more concrete usecases, but ideas for demo functionality would be useful.

Reply:
I added a textbox and some buttons that allow you to specify an account name regex to override the config for PortfolioList and generate an account listing for a string entered in the frontend.